pom.xml

```diff
@@ -306,6 +299,16 @@
 
     </dependencies>
 
+    <dependencyManagement>
```

Comment:
Why is it needed? BOM supplies it
https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml#L170-L174

Reply:
Not using bom.